build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--fetch-dependencies-only")
+      fetchDependencies "$@"
+      ;;
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
```

Comment:
It will be much easier to maintain, and it would allow the recipe to be run outside of docker. If you don't want `wget` in the docker image you can do `apt-get remove --purge wget && apt-get autoremove --purge` in the dockerfile after you've finished using it, but I probably wouldn't bother because it might come in handy later. Definitely use it to install, but it's up to you whether you keep it.